Source/UserManagement/Domain/StaffUser/Validators/AddStaffUserValidator.cs

```diff
@@ -46,26 +37,47 @@ public AddStaffUserValidator()
                     .IsInEnum()
                     .WithMessage("Preferred language is not correct - Has to be a supported languge");
 
-                //TODO: Validate MobilePhoneNumber based on localization
+                //TODO: Validate MobilePhoneNumbers based on localization
                 RuleFor(_ => _.MobilePhoneNumber)
                     .NotEmpty()
                     //.Matches(@"^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$")
                     .WithMessage("Mobile phone number is not correct - Has to be of correct form");
             });
 
+            When(_ => _.Role.RequiresAgeAndSex(), () =>
+            {
+                RuleFor(_ => _.YearOfBirth)
+                    .NotEmpty().WithMessage("YearOfBirth is not correct - Has to be specified");
+
+                RuleFor(_ => _.Sex)
```

Comment:
Be careful here.  An enum will have a default value if it is not passed (the MVC model builder will do this):